content/docs/04-clusters/06-cluster-management/05-backup-restore/01-add-backup-location.md

```diff
@@ -0,0 +1,357 @@
+	- Switch to the AWS console. Use the AWS account that hosts the S3 bucket.
+	- Navigate to the IAM service, and select **Roles** from the left **Main Menu**. 
+	- Click on the **Create role** button to create a new role. 
+	- Use the following configuration while creating the IAM role in the AWS console. 
+
+    |**AWS Console Field**|**Value**|
+    |---|---|
+    |Trusted entity type| AWS account.|
+    |An AWS account|Select the Another AWS account radio button.|
+    |AWS Account ID|Use the one displayed in Palette. It is Palette SaaS account ID.|
+    |Options|Select the Require external ID checkbox.|
+    |External ID|Use the one displayed in Palette. Palette generates the external ID.|
+    |Permissions policies| Attach the IAM policy defined above.|
+    |Role name|Your choice.|
+    |Role description|Your choice.|
+    
+	- Review the trusted entities and the attached permissions. 
+	- Finish creating the IAM role.
+	- Copy the IAM role ARN and paste it into Palette.
+
+  </Tabs.TabPane>
+
+	</Tabs>
+
+
+
+6. Click on the **Validate** button after providing the IAM user's access key or the IAM role ARN. 
+
+
+7. If the backup cloud account differs from the one where Palette deploys the clusters. In such a scenario, you must also authorize the latter cloud account. You must update the existing trust policy by appending the following permission. The permission below will authorize the AWS account where Palette deploys the clusters to assume the current IAM role. <br /> <br />	
+
+	```json
+	{
+		"Effect": "Allow",
+		"Principal": {
+				"AWS": "arn:aws:iam::[AWS-ACCOUNT-ID-WHERE-PALETTE-DEPLOYS-THE-CLUSTERS]:root"
+		},
+		"Action": "sts:AssumeRole"
+	}
+	```
+	If you want to establish a trust relationship with a specific IAM role, say *SpectroCloudRole*, located within the AWS account that hosts your clusters. You can use the `"arn:aws:iam::[YOUR-AWS-ACCOUNT-ID-WHERE-PALETTE-DEPLOYS-THE-CLUSTERS]:role/SpectroCloudRole"` ARN instead. Your final trust policy will become similar to the policy defined below. It has two trust relationships, one for the Palette SaaS account and another for the AWS account that hosts your clusters. <br /> <br />
+
+  ```json
+  {
+    "Version": "2012-10-17",
+    "Statement": [
+      {
+        "Effect": "Allow",
+        "Principal": {
+            "AWS": "arn:aws:iam::[AWS-ACCOUNT-ID-OF-PALETTE-SaaS]:root"
+        },
+        "Action": "sts:AssumeRole",
+        "Condition": {
+            "StringEquals": {
+                "sts:ExternalId": "[YOUR-EXTERNAL-ID]"
+            }
+        }
+      }
+      {
+        "Effect": "Allow",
+        "Principal": {
+            "AWS": "arn:aws:iam::[AWS-ACCOUNT-ID-WHERE-PALETTE-DEPLOYS-THE-CLUSTERS]:root"
+        },
+        "Action": "sts:AssumeRole"
+      }
+    ]
+  }
+  ```
+
+8. Click **Create** to finish the location creation wizard. 
```

Comment:
These two steps did not make sense when I read them. It's not clear what accounts trust policy and IAM entity is being changed. Here is a lil advice, if you are dealing with two similar entities, label one them, you can say account A, and account B. As you explain each accounts IAM entities and trust policies you can say, update account A's trust policy, etc.  But in its current form, this is difficult to follow.